unit_test/sample_key/readme.txt

```diff
@@ -145,15 +181,27 @@ openssl pkey -in end_requester.key -inform PEM -pubout -outform PEM -out end_req
 openssl pkey -in end_requester.key -inform PEM -pubout -outform DER -out end_requester.key.pub.der
 openssl pkey -in end_responder.key -inform PEM -pubout -outform PEM -out end_responder.key.pub
 openssl pkey -in end_responder.key -inform PEM -pubout -outform DER -out end_responder.key.pub.der
+openssl x509 -req -in end_requester.req -out end_requester_with_spdm_req_rsp_eku.cert -CA inter.cert -CAkey inter.key -sha256 -days 3650 -set_serial 4 -extensions v3_end_with_spdm_req_rsp_eku -extfile ../openssl.cnf
+openssl x509 -req -in end_requester.req -out end_requester_with_spdm_req_eku.cert -CA inter.cert -CAkey inter.key -sha256 -days 3650 -set_serial 5 -extensions v3_end_with_spdm_req_eku -extfile ../openssl.cnf
+openssl x509 -req -in end_requester.req -out end_requester_with_spdm_rsp_eku.cert -CA inter.cert -CAkey inter.key -sha256 -days 3650 -set_serial 6 -extensions v3_end_with_spdm_rsp_eku -extfile ../openssl.cnf
+openssl asn1parse -in end_requester_with_spdm_req_rsp_eku.cert -out end_requester_with_spdm_req_rsp_eku.cert.der
+openssl asn1parse -in end_requester_with_spdm_req_eku.cert -out end_requester_with_spdm_req_eku.cert.der
+openssl asn1parse -in end_requester_with_spdm_rsp_eku.cert -out end_requester_with_spdm_rsp_eku.cert.der
+openssl x509 -req -in end_responder.req -out end_responder_with_spdm_req_rsp_eku.cert -CA inter.cert -CAkey inter.key -sha256 -days 3650 -set_serial 7 -extensions v3_end_with_spdm_req_rsp_eku -extfile ../openssl.cnf
+openssl x509 -req -in end_responder.req -out end_responder_with_spdm_req_eku.cert -CA inter.cert -CAkey inter.key -sha256 -days 3650 -set_serial 8 -extensions v3_end_with_spdm_req_eku -extfile ../openssl.cnf
+openssl x509 -req -in end_responder.req -out end_responder_with_spdm_rsp_eku.cert -CA inter.cert -CAkey inter.key -sha256 -days 3650 -set_serial 9 -extensions v3_end_with_spdm_rsp_eku -extfile ../openssl.cnf
+openssl asn1parse -in end_responder_with_spdm_req_rsp_eku.cert -out end_responder_with_spdm_req_rsp_eku.cert.der
+openssl asn1parse -in end_responder_with_spdm_req_eku.cert -out end_responder_with_spdm_req_eku.cert.der
+openssl asn1parse -in end_responder_with_spdm_rsp_eku.cert -out end_responder_with_spdm_rsp_eku.cert.der
 popd
 
 pushd ecp384
 openssl genpkey -genparam -out param.pem -algorithm EC -pkeyopt ec_paramgen_curve:P-384
-openssl req -nodes -x509 -days 3650 -newkey ec:param.pem -keyout ca.key -out ca.cert -sha384 -subj "//CN=DMTF libspdm ECP256 CA"
+openssl req -nodes -x509 -days 3650 -newkey ec:param.pem -keyout ca.key -out ca.cert -sha384 -subj "//CN=DMTF libspdm ECP384 CA"

```

Comment:
The typo in ECC-P384 and ECC-P521 certs subject are fixed by the way,
and key pair and certs for such 2 algorithms are re-generated to use correct subject name.